### DISCUSSION TITLE
Action button like Submit , create , create and other   . how to rename it .

### DISCUSSION THREAD

**Discussion by Raghu135** [filamentphp/filament#13458] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Action button like Submit , create , create and other   . action button like these are unable to rename . Hope someone could solve the issue

**Comment by leandrocfe** [comment]
Publish translations and edit the labels

https://filamentphp.com/docs/3.x/panels/installation#publishing-translations
